pkg/testing/v1/configuration.go

```diff
@@ -108,6 +108,16 @@ func WithConfigLabel(key, value string) ConfigOption {
 	}
 }
 
+// WithConfigAnn attaches a particular label to the configuration.
+func WithConfigAnn(key, value string) ConfigOption {
+	return func(config *v1.Configuration) {
+		if config.Annotations == nil {
+			config.Annotations = make(map[string]string, 1)
+		}
+		config.Annotations[key] = value
+	}
```

Comment:
```suggestion
		config.Annotations = kmeta.UnionMaps(config.Annotations, map[string]string{key,value})
	}
```

is a code golf alternative.

Reply:
*golf clap*

Normally I'd try to put that in my other 'nits' PR, but here it is